### ISSUE TITLE
[Accessibility]: axis tooltip should be hidden when user presses Escape for HEATMAP

### ISSUE THREAD

**Issue by Akshat55** [carbon-design-system/carbon-charts#1453]
### Environment

Browser

### What happened? What did you expect to happen instead?

When navigating the axis values using tab, tooltip covers content due to its automatic position near the target. Users should be able to `Escape` the tooltip to dismiss it.

### What WCAG 2.1 checkpoint does the issue violate?

NA

### Version

@carbon/charts@1.6.0
